parsing/lexer.mll

```diff
@@ -377,7 +378,11 @@ rule token = parse
         COMMENT (s, loc) }
   | "(**"
       { let s, loc = with_comment_buffer comment lexbuf in
-        DOCSTRING (Docstrings.docstring s loc) }
+        if !handle_docstrings then
+          DOCSTRING (Docstrings.docstring s loc)
+        else
+          COMMENT (s, loc)
```

Comment:
Right. Interaction with external tools (that would purposefully deactivate the special handling of docstring in the lexer) is certainly a good reason to add back the leading '*' start. (It might also be useful to check in the test suite that no comments are lost in cmt files.)